server/backend/api/app/crud/crud_application.py

```diff
@@ -29,6 +31,35 @@ def get_applications(db: Session):
     return fam_apps
 
 
+def get_applications_by_granted_apps(db: Session, access_roles: List[str]) -> List[models.FamApplication]:
+    """ Get applications based on access roles that are associated with.
+        Note, this isn't to find the applications that the 'roles belong to'.
+
+    :param access_roles: Cognito token custom groups (aka the Role(s) for FAM).
+    :return: list of applications granted to view.
+    """
+    LOGGER.debug(f"Running get_applications_by_granted_app, access_roles: {access_roles}")
+
+    # Convert to Upper Case.
+    REPLACE_STR_KEY = "_ACCESS_ADMIN"
+    upper_app_names = crud_utils.replace_str_list(crud_utils.to_upper(access_roles), REPLACE_STR_KEY, "")
```

Comment:
Yes, after reviewing below, it is possible for a role like FOM_DEV_REVIEWER to be passed in, but this code assumes this doesn't happen.

Reply:
jwt_validation "authorize" assume for administrator we have this "X_ACCESS_ADMIN" format or it won't even pass the function validation and endpoints won't get called. That's the current endpoints security. So for "FOM_DEV_REVIEWER" user, it can't access FAM endpoints at all. But maybe you have other thought?

Reply:
A user like myself can have both FOM_DEV_ACCESS_ADMIN and FOM_DEV_REVIEWER roles assigned, so the access_roles list will include both.

Reply:
Thanks Basil, understood your concerns now. Added the filter to find only access admin role first.